### DISCUSSION TITLE
Does the value of ‘sel’ must be proportional to the number of atoms？

### DISCUSSION THREAD

**Discussion by phyoung123** [deepmodeling/dpgen#740] [Q&A] [answered]
There is a file, out.json, auto-generated by the program transformed from the input file when the program onset to tain the model, I set 'sel' to 'auto' in the input file, but it is assigned a value of 40 40 108, while the atomic ratio is 1:1:3.  Does its value have to be proportional to the number of atoms?

**Comment by AnguseZhang** [comment, marked as answer]
No, it's not a must to set `sel` proportional to the number of atoms.
See https://docs.deepmodeling.com/projects/deepmd/en/master/troubleshooting/howtoset_sel.html?highlight=sel

**Reply by phyoung123** [reply]
Thanks
